contributing/getting-started-e2e.md

```diff
@@ -412,21 +412,34 @@ GitHub offers a convenient "Delete branch" button on the PR page after the chang
 
 ```
 # go back to the master branch
-git checkout master 
+git checkout master
 
 # delete the branch in your local repository
 git branch -D <branch name>
 
 # delete the branch in your GitHub fork (if you didn't use the UI)
-git push origin --delete <branch name> 
+git push origin --delete <branch name>
 ```
 
-# Celebrate
+# See your changes in production
+
+**Congratulations on making your first change to AMP!**
+
+If your change affected internal documentation, tests, the build process, etc. you can generally see your changes right after they're merged.  If your change was to the code that runs on AMP pages across the web you'll have to wait for the change to be included in a release.
+
+In general we cut a release of amphtml on Thursdays during working hours (Pacific time) and push it to the AMP Dev Channel that day.  After verifying there are no issues, we start rolling that build out the following Monday and complete the push that Thursday.
+
+You can see whether your change made it into the Dev Channel or production build on the [amphtml Releases page](https://github.com/ampproject/amphtml/releases).  The build marked `Draft` is the version on the Dev Channel and the build marked `Latest Release` is what is running in production.  Your Pull Request will be listed in the first build that includes it; if you don't see your Pull Request listed it will likely be in the next build.
```

Comment:
Got it; I was trying to reverse engineer the labels from where the Releases page was and what was currently on canary.  I talked to @erwinmombay about it and realized they temporarily didn't have the right labels. :)